sklearn/preprocessing/data.py

```diff
@@ -229,11 +233,17 @@ class MinMaxScaler(BaseEstimator, TransformerMixin):
 
     scale_ : ndarray, shape (n_features,)
         Per feature relative scaling of the data.
+
+
+
     """
 
-    def __init__(self, feature_range=(0, 1), copy=True):
+    def __init__(self, feature_range=(0, 1), copy=True, selected="all"):
         self.feature_range = feature_range
         self.copy = copy
+        self.selected = selected
+        self.scaled_features_ = None if selected == "all" else selected
```

Comment:
There should be no logic in the init: this logic should be moved to fit (the goal is to be robust to modification of parameters in an instanciated object).